src/components/layer-controller/LayerOptions.js

```diff
@@ -42,6 +42,35 @@ function OpacitySlider({ value, handleChange }) {
   );
 }
 
+function GlobalSelectionSlider({
+  field,
+  value,
+  handleChange,
+  possibleValues,
+}) {
+  return (
+    <Slider
+      value={value}
+      // See https://github.com/hubmapconsortium/vitessce-image-viewer/issues/176 for why
+      // we have the two handlers.
+      onChange={
+        (event, newValue) => handleChange({ selection: { [field]: newValue }, event })
+      }
+      onChangeCommitted={
+        (event, newValue) => handleChange({ selection: { [field]: newValue }, event })
+      }
```

Comment:
re: above, newValue should be an integer index. That way it's easy to switch between slider and dropdown,